src/generator/operations.ts

```diff
@@ -225,7 +224,7 @@ function getZeroReturnValue(op: Operation, apiType: 'api' | 'op' | 'handler'): s
   } else if (hasSchemaResponse(op)) {
     // simple schema response
     returnType = `${(<SchemaResponse>op.responses![0]).schema.language.go!.responseType.name}{}`;
-  } else if (op.language.go!.headAsBoolean) {
+  } else if (op.language.go!.headAsBoolean === true) {
```

Comment:
Why does this have to explicitly check if it's equal to true?

Reply:
IIRC the former will return true if the key `headAsBoolean` exists, it doesn't check its value.  It's a bit pedantic since we only ever set this to `true`, but could lead to bugs in the odd case we decide to set it to `false`.

Reply:
Ok, sounds good 👍